### PR TITLE
SUBMARINE-309. Make yarn service configuraiton optional in installation guide.

### PR DESCRIPTION
**Pull request by yuanzac**
### What is this PR for?
At this time, Submarine has two runtimes for yarn clusters, yarn service and tony. So the installation guide should work for both two rumtimes. As the installation of calico and registrydns is only for yarn service, they should be marked as optional. 

### What type of PR is it?
Improvement

### Todos
* [ ] - Task

### What is the Jira issue?
https://issues.apache.org/jira/browse/SUBMARINE-309

### How should this be tested?
https://travis-ci.org/yuanzac/hadoop-submarine/builds/627688854?utm_source=github_status&utm_medium=notification

### Questions:
* Does the licenses files need update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? Yes

### CODE REVIEW

File: dev-support/submarine-installer/InstallationGuide.md [hunk 1]
```diff
@@ -599,3 +520,185 @@ Add configurations in container-executor.cfg
    root=/sys/fs/cgroup
    yarn-hierarchy=/hadoop-yarn
    ```
+
+## Yarn Service Runtime Requirement (Deprecated)
```
Comment (liuxunorg) [inline dev-support/submarine-installer/InstallationGuide.md]: Change `Deprecated` to `Not Recommend`?

File: dev-support/submarine-installer/InstallationGuideChineseVersion.md [hunk 2]
```diff
@@ -614,6 +539,182 @@ Distributed-shell + GPU + cgroup
  --worker_launch_cmd "python /test/cifar10_estimator/cifar10_main.py --data-dir=hdfs://${dfs_name_service}/tmp/cifar-10-data --job-dir=hdfs://${dfs_name_service}/tmp/cifar-10-jobdir --train-steps=500 --eval-batch-size=16 --train-batch-size=16 --sync --num-gpus=1"
 ```
 
+## Yarn Service Runtime  (Deprecated)
```
Comment (liuxunorg) [inline dev-support/submarine-installer/InstallationGuideChineseVersion.md]: ## Yarn Service Runtime  (不推荐)

**Comment by yuanzac** [comment]
@liuxunorg，
Thanks for the review. Just update the PR.

File: dev-support/submarine-installer/InstallationGuideChineseVersion.md [hunk 3]
```diff
@@ -368,83 +368,16 @@ $ python >> tf.__version__
...
-
 ## 安装 Hadoop
 
-### 编译 Hadoop
-
-```
-mvn package -Pdist -DskipTests -Dtar
-```
+### 安装 Hadoop
+首先，我们通过源码编译或者直接从官网[Hadoop Homepage](https://hadoop.apache.org/)下载获取hadoop包。
```
Comment (liuxunorg) [inline dev-support/submarine-installer/InstallationGuideChineseVersion.md]: `下载获取hadoop包。` Add a space before and after `hadoop`

Comment (liuxunorg) [inline dev-support/submarine-installer/InstallationGuideChineseVersion.md]: `官网[Hadoop Homepage]` Add a space before `[Hadoop Homepage]`

File: dev-support/submarine-installer/InstallationGuideChineseVersion.md [hunk 4]
```diff
@@ -614,6 +540,182 @@ Distributed-shell + GPU + cgroup
...
+member 85ffe9aafb7745cc is healthy: got healthy result from http://${etcd_host_ip2}:2379
+member b3d05464c356441a is healthy: got healthy result from http://${etcd_host_ip3}:2379
+cluster is healthy
+
+$ etcdctl member list
+3adf2673436aa824: name=etcdnode3 peerURLs=http://${etcd_host_ip1}:2380 clientURLs=http://${etcd_host_ip1}:2379 isLeader=false
+85ffe9aafb7745cc: name=etcdnode2 peerURLs=http://${etcd_host_ip2}:2380 clientURLs=http://${etcd_host_ip2}:2379 isLeader=false
+b3d05464c356441a: name=etcdnode1 peerURLs=http://${etcd_host_ip3}:2380 clientURLs=http://${etcd_host_ip3}:2379 isLeader=true
+```
+其中，${etcd_host_ip*} 是etcd服务器的ip
```
Comment (liuxunorg) [inline dev-support/submarine-installer/InstallationGuideChineseVersion.md]: 是etcd服务器的ip
Add a space before and after `etcd`.
`ip` need this too.

**Comment by yuanzac** [comment]
@liuxunorg,
Thanks a lot. Just updated this PR.

**Comment by liuxunorg** [comment]
Will merge if no more comment

**Pull request closed without merging**